shared/types/node.ts

```diff
@@ -34,5 +34,11 @@ export interface NodeConfig {
     'network-id': number;
     [k: string]: any;
   };
+  main: {
+    'genesis-time': string;
+    'layer-duration-sec': number;
+    'layers-per-epoch': number;
+    [k: string]: any;
```

Comment:
is it possible to define other properties and remove this k:string]: any ? it is painful way to write a code